pywinauto/unittests/test_Calendar.py

```diff
@@ -99,5 +100,8 @@ def assertThatSystemTimeIsEqualCurrentDateTime(self,systemTime, now):
         self.assertEqual(systemTime.wMonth, now.month)
         self.assertEqual(systemTime.wDay, now.day)
 
+    def set_calendar_state_into_multiselect(self):
```

Comment:
This method is called only once and it's one line. I think it should not be a method. Just call `self.app['Common Controls Sample']['MCS_MULTISELECT'].Click()` in the test. `WrapperObject()` is called implicitly, so you don't need it in production code.